### PR TITLE
Basic faq docs

### LINKED ISSUE #177: Add FAQ style document to documentation
Can we write up a little bit of documentation for how scientists should expect to work with the [CMIP data platform](https://www2.cisl.ucar.edu/resources/cmip-analysis-platform) and the intake-esm catalogs? An FAQ style document would be helpful.

Questions like:

1) How could I create my own catalog?
2) Where is the catalog for the CESM data not in CMIP6?
3) How long does it take for a data request to the platform to show up in the catalog?
4) If I request data from the platform and it isn’t availble yet, do I need to go back regularly to keep asking for it

 @jhamman, thank you for bringing attention to this issue

### PR DESCRIPTION
**Pull request by jhamman**
closes #177 

@andersy005 please review and add relevant details as you see fit.

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Is there a list of existing catalogs?
+-------------------------------------
+
+The table below is an incomplete list of existing catalogs.
+Please feel free to add to this list.
+
++-------------------+--------------------+-----------------+
+| Catalog Name      | Platform           | Details         |
++===================+====================+=================+
+| CMIP6-GLADE       | NCAR-Cheyenne      | TBD             |
```
Comment (jhamman) [inline docs/source/faq.rst]: some of these should link to the CMIP AP page. Sphinx was not agreeing with me on the reference so I gave up.

File: docs/source/cmip_ap.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+`NCAR's CMIP Analysis Platform (CMIP AP) <https://www2.cisl.ucar.edu/resources/cmip-analysis-platform>`_
+inncludes a large collection of CMIP5 and CMIP6 data sets. 
+
+Requesting data sets
+--------------------
+
+`Use this form <https://www2.cisl.ucar.edu/resources/cmip-analysis-platform/request-cmip6-data-sets>`_
+to request new data be added to the CMIP AP. Typically requests are fulfilled
+within two weeks. Contact `CISL <https://www2.cisl.ucar.edu/user-support/getting-help>`_
+if you have further questions. Intake-ESM catalogs are regularly updated
```
Comment (jhamman) [inline docs/source/cmip_ap.rst]: Note the ambiguity here. Would be useful to have this clarified on the CMIP AP webpage.

**Comment by andersy005** [comment]
@jhamman, thank you for getting this started!

File: docs/source/faq.rst (hunk 1 above)
Comment (andersy005) [reply docs/source/faq.rst]: By `CMIP AP page`, do you mean https://www2.cisl.ucar.edu/resources/cmip-analysis-platform or the cmip_ap.rst document?

File: docs/source/faq.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Is there a list of existing catalogs?
+-------------------------------------
+
+The table below is an incomplete list of existing catalogs.
+Please feel free to add to this list or raise an issue on `GitHub <https://github.com/NCAR/intake-esm/issues/new>`_.
+
++-------------------------+-----------------+-----------------+
+| Catalog Name            | Platform        | Details         |
++=========================+=================+=================+
+| `CMIP6-GLADE`_          | NCAR-Cheyenne   | TBD             |
```
Comment (andersy005) [inline docs/source/faq.rst]: What are the appropriate entries for the `Details` column?

Comment (jhamman) [reply docs/source/faq.rst]: I think we should link to where you can find out more about each catalog. For the GLADE catalogs, this is probably best directed to the `cmip_ap.rst` page.

**Comment by jhamman** [comment]
@andersy005 - looks good. I think it would be worth adding a (optional?) link to each datasets documentation page. Somewhere where users can learn more about the dataset, citation, etc.

**Comment by andersy005** [comment]
> I think it would be worth adding a (optional?) link to each datasets documentation page. Somewhere where users can learn more about the dataset, citation, etc.

👍 I just added this option. When you get time, can you update some of the entries? I wasn't sure of what to link to for some datasets (such as CMIP5/6).

**Pull request merged**